firefly_client/firefly_client.py

```diff
@@ -1413,28 +1416,93 @@ def set_stretch(self, plot_id, stype=None, algorithm=None, **additional_params):
         else:
             serialized_rv = self._create_rangevalues_standard(algorithm, stype, **additional_params)
 
-        st_data = [{'band': 'NO_BAND', 'rv': serialized_rv, 'bandVisible': True}]
+        bands_3color = ['RED', 'GREEN', 'BLUE', 'ALL']
+        if (not band):
+            band_list = ['NO_BAND']
+        elif band in bands_3color:
+            band_list = ['RED', 'GREEN', 'BLUE'] if band == 'ALL' else [band]
+        else:
+            raise ValueError('invalid band: %s' % band)
+
+        st_data = []
+        for b in band_list:
+            st_data.append({'band': b, 'rv': serialized_rv, 'bandVisible': True})
+
         payload = {'stretchData': st_data, 'plotId': plot_id}
 
         return_val = self.dispatch_remote_action_by_post(self.channel, FireflyClient.ACTION_DICT['StretchImage'], payload)
         return_val['rv_string'] = serialized_rv
         return return_val
 
+    def set_stretch_hprgb(self, plot_id, asinh_q_value=None, scaling_k=1.0,
+                          pedestal_value=1, pedestal_type='percent'):
+        """
+        Change the stretch of RGB image (hue-preserving rgb case). When a parameter is list,
+        the first value if for red, the second value - for green, and the third value - for blue band.
+        Otherwise the same value is used for all three bands.
+
+        Parameters
+        ----------
+        plot_id : `str` or `list` of `str`
+            ID of the plot to be stretched. If `plot_id` is a list or tuple, then each plot in the list
+            or the tuple is stretched in order.
+        asinh_q_value : `float`, optional
+            The asinh softening parameter for Asinh stretch.
+            Use Q=0 for linear stretch, increase Q to make brighter features visible.
+            When not specified, Q is calculated by Firefly to use full color range for intensity.
+        scaling_k : `float` or `list` of `float`, optional
+            Scaling coefficient from 0.1 to 10 (the default is 1).
+        pedestal_type : {'percent','minmax','absolute','zscale','sigma'} or `list` of `str`, optional
```

Comment:
```suggestion
        pedestal_type : {'percent', 'minmax', 'absolute', 'zscale', 'sigma'} or `list` of `str`, optional
```

Spaces after the commas, will make the Sphinx output look better, I think.